winlogbeat/etc/fields.yml

```diff
@@ -228,7 +228,7 @@ eventlog:
       description: The version number of the event's definition.
 
     - name: xml
-      type: string
```

Comment:
Having lots of huge unique "keywords" is also an anti-pattern, and probably not useful. I'd rather not raise the 1024 limit.